src/type-utils.ts

```diff
@@ -48,20 +69,20 @@ const checkRequired = <T>(validateFn: (value: any) => T | Promise<T>) => {
 };
 
 const equality = {
-	'is equal to': (from: string, to: string) => ['Equals', from, to],
-};
+	'is equal to': (from, to): EqualsNode => ['Equals', from, to],
+} satisfies NativeFactTypes[string];
 export const nativeFactTypeTemplates = {
 	equality,
 	comparison: {
-		'is less than': (from: string, to: string) => ['LessThan', from, to],
-		'is less than or equal to': (from: string, to: string) => [
+		'is less than': (from, to): LessThanNode => ['LessThan', from, to],
+		'is less than or equal to': (from, to): LessThanOrEqualNode => [
 			'LessThanOrEqual',
 			from,
 			to,
 		],
 		...equality,
 	},
-};
+} satisfies Record<string, NativeFactTypes[string]>;
```

Comment:
Can't this be `satisfies NativeFactTypes;` ?

Reply:
It could but that's not accurate to what it is, because it is not itself a `NativeFactTypes` but rather a dictionary of fragments of them, so I made the typing in such a way as to try to represent that difference